docs/python/environments.md

```diff
@@ -18,34 +18,38 @@ This article discusses the helpful Python environments features available in Vis
 
 ### Global environments
 
-By default, any Python interpreter installed runs in its own **global environment**. They aren't specific to a particular project. For example, if you just run `python`, `python3`, or `py` at a new terminal (depending on how you installed Python), you're running in that interpreter's global environment. Any packages that you install or uninstall affect the global environment and all programs that you run within it.
+By default, any Python interpreter installed runs in its own **global environment**. For example, if you just run `python`, `python3`, or `py` at a new terminal (depending on how you installed Python), you're running in that interpreter's global environment. Any packages that you install or uninstall affect the global environment and all programs that you run within it.
 
-Working in the global environment is an easy way to get started. If you install packages in that environment, though, in time it will become crowded and make it difficult to properly test an application.
+Do note that if you install packages into your global environment, though, in time it will become crowded with disparate packages and make it difficult to properly test an application. You typically want to create an environment for each workspace.
```

Comment:
Is "disparate" the right word?

Reply:
Somewhat because the problem is mixing and matching of things that have no relation to each other. But it's fair that the word may not be well-known to non-native speakers, so I've simplified the wording.